app/src/main/java/org/dicekeys/app/repositories/DiceKeyRepository.kt

```diff
@@ -0,0 +1,30 @@
+package org.dicekeys.app.repositories
+
+import org.dicekeys.app.encryption.EncryptedDiceKey
+import org.dicekeys.dicekey.DiceKey
+
+/*
+    A Repo to store DiceKeys. DiceKeys can be only in memory or backed by encrypted storage
```

Comment:
So this is an in-memory store that effectively abstracts away a map.

Since we only have 1 DiceKey loaded at a time, a map may be more than we need.

Reply:
The idea was that you can have more than one DiceKeys unlocked at a time.
If this functionality is not needed, i can simplify it.

Reply:
Might come in handy at some point. As long as you understand current needs and document that it's overkill, okay to leave as is.